### PR TITLE
adds extender with tolerations for ksa region

### PR DESCRIPTION
**Pull request by Disper**
**Description**

Changes proposed in this pull request:

- implements #299 
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Comment by Disper** [comment]
verified on DEV 
image

File: internal/gardener/shoot/extender/tolerations_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"testing"
+
+	imv1 "github.com/kyma-project/infrastructure-manager/api/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestTolerationsExtender(t *testing.T) {
+	t.Run("Extend Tolerations", func(t *testing.T) {
```
Comment (koala7659) [inline internal/gardener/shoot/extender/tolerations_test.go:14]: I think we should have negative test case here - this key should not be added for any other region than "me-central2"

**Review by koala7659 — CHANGES_REQUESTED** [review]
Please see my comment and add missing test case

**Review by koala7659 — APPROVED** [review]
LGTM

**Pull request merged**